### DISCUSSION TITLE
How to center a dropdown on top of a component?

### DISCUSSION THREAD

**Discussion by dev0T** [saadeghi/daisyui#1740] [Q&A] [answered]
I am attempting to use a dropdown as a popover with an image inside it.

How do I have it centred with the component below it?

**Comment by saadeghi** [comment, marked as answer]
You can horizontally center it using `left-1/2 -translate-x-1/2` classnames:  
https://play.tailwindcss.com/Nnt1D06qAP

However I would suggest using a JS library like [Floating UI](https://floating-ui.com/) for popovers because a JS library can handle the position dynamically, can avoid z-index issues, and can handle scrolling, keyboard navigation, etc.

**Reply by dev0T** [reply]
Thank you very much! I have been looking into it, but using CSS is now enough for my use case. Appreciate it.
